### DISCUSSION TITLE
Express Hibernate Queries as Type-Safe Java Streams

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#31078] [Quarkus Blog/Website]
# Express Hibernate Queries as Type-Safe Java Streams

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/jpastreamer-extension/

**Comment by ahndmal** [comment]
Thank you for the article!

**Comment by allixus** [comment]
Is this usable in a reactive context or is the query creation synchronous and blocking (like JPA's CriteriaBuilder)?
